### DISCUSSION TITLE
Need Add Environment for timezone

### DISCUSSION THREAD

**Discussion by nixk99** [juanluisbaptiste/docker-postfix#62] [Ideas]
Its better, add environment for timezone

**Comment by juanluisbaptiste** [comment]
Hi,

There is no postfix configuration to set the timezone, it is a OS configuration. If what you need is to set the container's timezone to your local timezone you can do it by mounting `/etc/localtime` and `/etc/timezone` into the container:

```
volumes:
- "/etc/timezone:/etc/timezone:ro"
- "/etc/localtime:/etc/localtime:ro"
```
